tests/unit/test_cache_scope_after_loop.py

```
@@ -0,0 +1,71 @@
+from flujo.application.core.policies.cache_policy import DefaultCacheStepExecutor
+from flujo.steps.cache_step import CacheStep
+from flujo.domain.dsl import Step
+from flujo.domain.models import StepResult, Success
+from flujo.application.core.types import ExecutionFrame
+
+
+class _FakeCacheBackend:
+    def __init__(self) -> None:
+        self.get_calls = 0
+
+    async def get(self, key: str):
```

Ruff (0.14.7)

Unused method argument: `key`

(ARG002)